client/src/utils.js

```diff
@@ -115,6 +115,10 @@ export default {
 			return true
 		}
 	},
+
+	getForeignKey: function(obj) {
```

Comment:
Sorry to nitpick, but getForeignKey sounds too SQL-ish. Can we call it something else, like `getReference` or something?